src/test/java/page/GoogleHomePage.java

```diff
@@ -0,0 +1,36 @@
+package page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleHomePage extends BasePage {
+
+    private static final String GOOGLE_HOMEPAGE_URL = "https://www.google.com/";
+    private WebDriver driver;
+
+    @FindBy(name = "q")
+    private WebElement googleSearchField;
+
+    @FindBy(name = "btnK")
+    private WebElement googleSearchButton;
+
+    public GoogleHomePage(WebDriver driver) {
+        this.driver = driver;
```

Comment:
all you need in your constructor is just `super(driver);`

Reply:
Done, thanks